Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -2985,6 +2990,9 @@ private void setActivityLifeCycleObserver(Application application) {
 
         @Override
         public void onActivityCreated(Activity activity, Bundle bundle) {
+            if(PromoViewHandler.getInstance().isInstallOrOpenPromoPending()){
+                PromoViewHandler.getInstance().showPendingPromoView(activity);
```

Comment:
since this is for install / open cases - do we want to limit this to the first activity, or all activity creates OK?